Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,14 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  for (let i = 0; i < arr.length; i++) {
+    for (let j = 0; j < i; j++) {
+      if (arr[i] === arr[j]) {
+        arr.splice(i, 1);
```

Comment:
That's Smart! 🥇 

I think it's time to move to a bit more advanced level, what about reading:
https://gomakethings.com/removing-duplicates-from-an-array-with-vanilla-javascript/

It's super useful to learn new techniques, Skip what you find complicated!